aten/src/ATen/native/native_functions.yaml

```diff
@@ -129,10 +129,12 @@
   variants: method
 
 - func: _use_cudnn_ctc_loss(Tensor log_probs, Tensor targets, int[] input_lengths, int[] target_lengths, int blank) -> bool
+  device_guard: False
   dispatch:
     CUDA: _use_cudnn_ctc_loss
 
 - func: _cudnn_ctc_loss(Tensor log_probs, Tensor targets, int[] input_lengths, int[] target_lengths, int blank, bool deterministic, bool zero_infinity) -> (Tensor, Tensor)
+  device_guard: False
```

Comment:
`log_probs` and `targets` are expected on different device:

https://github.com/pytorch/pytorch/blob/20d7916a6a4dd2577521ece0e6970caeaac512c8/aten/src/ATen/native/cudnn/LossCTC.cpp#L80-L81

Will need to add device guard there.

Reply:
BTW, if you're unify device check and guard, then the old key name `device_guard` is a misnomer as you aren't just doing guarding, you're also doing checking. Perhaps `device_check_and_guard` or `device_common`?

Reply:
what about `common_device_guard` ;)